qa/workunits/cephadm/test_iscsi_pids_limit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -ex
+
+# checks if the containers default pids-limit (4096) is removed and Iscsi
+# containers continue to run
+# exits 1 if fails
+
+ISCSI_CONT_IDS=$(podman ps -qa --filter='name=iscsi')
```

Comment:
podman isn't available on ubuntu focal (20.04).  I think you would need something like
```
if which podman 2>&1 >/dev/null; then
    CONT_CMD="podman"
else
    CONT_CMD="docker"
fi
```
or constrain this job to centos.